initialscreen.css

```diff
@@ -0,0 +1,635 @@
+* {
+    Box-sizing:  border-box;
+    font-family:'Raleway', sans-serif;
+    font-weight: lighter;
+}
+html {
+    scroll-behavior: smooth;
+}
+body {
+    background-color: #E5EBFF;
+    margin: 0;
+}
+.headerbox {
+    height: 100vh;
+    background-image: url("beautiful-cold-dawn-547115.jpg");
+    width: 100%;
+    background-size: cover;
+    background-position: center;
+}
+.applicationimages1, .applicationimages2, .descriptionbox {
+    float: left;
+    width: 33.33%;
+    height: 100%;
+    margin: auto;
+}
+.logoleft {
+    width: 23%;
+    height: auto;
+    padding: 3%;
+    float: left;
+}
+.logoright {
+    width: 25%;
+    height: auto;
+    padding: 3%;
+    float: right;
+}
+.descriptionbox {
+    text-align: center;
+    position: relative;
+}
+.descrip-holder {
+    position: absolute;
+    width: 100%;
+    height: 50%;
+    margin-top: 10%;
+    margin-bottom: 50%;
+}
+.name {
+    font-size: 3vw;
+    color: #CCFBFE;
+    font-weight: initial;
+    border-style: solid;
+    border-width: 0 0 0.1rem 0;
```

Comment:
Avoid using rem for anything other than font size

Reply:
every rem is gone now thanks.